bundles/mapping/mapmodule/service/map.layer.js

```diff
@@ -75,6 +76,7 @@ Oskari.clazz.define('Oskari.mapframework.service.MapLayerService',
         };
 
         Oskari.makeObservable(this);
+        this.stateListeners = [];
```

Comment:
Remove `stateListeners`, the service is already observable.

Reply:
Done